test/scalapack_wrappers.hh

```diff
@@ -4634,31 +4634,31 @@ void scalapack_pgecon(
 extern "C" {
 
 void scalapack_pspocon(
-    const char* uplo, const char* norm, blas_int* n,
+    const char* uplo, blas_int* n,
```

Comment:
I do not remember why the `norm` has disappeared.

Reply:
It was a mistake. `gecon` takes the norm, which can be the 1-norm or Inf-norm. Since the 1-norm and Inf-norm are the same for a Hermitian matrix, no need to specify it in `pocon`.